From 0263750a0db723760d61bcaafc6964a371adcdfc Mon Sep 17 00:00:00 2001
From: Diego Biurrun <diego@biurrun.de>
Date: Mon, 25 Aug 2014 17:26:15 +0200
Subject: [PATCH 19/37] vfwcap: Add fallback define for HWND_MESSAGE

Some obsolete versions of the MinGW32 runtime (<4.0.0) lack the definition.

(cherry picked from commit ab56fabe6294524e99815451ad01e4ff50c6d734)
Signed-off-by: Diego Biurrun <diego@biurrun.de>
---
 libavdevice/vfwcap.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/libavdevice/vfwcap.c b/libavdevice/vfwcap.c
index c067be3..b47de1b 100644
--- a/libavdevice/vfwcap.c
+++ b/libavdevice/vfwcap.c
@@ -27,6 +27,11 @@
 #include <windows.h>
 #include <vfw.h>
 
+/* Some obsolete versions of MinGW32 before 4.0.0 lack this. */
+#ifndef HWND_MESSAGE
+#define HWND_MESSAGE ((HWND) -3)
+#endif
+
 struct vfw_ctx {
     const AVClass *class;
     HWND hwnd;
-- 
1.9.1

